integration-tests/bats/import-update-tables.bats

```diff
@@ -1017,3 +1017,149 @@ DELIM
     [ "$status" -eq 1 ]
     [[ "$output" =~ "All constraints are not satisfied" ]] || false
 }
+
+@test "import-update-tables: bit types" {
+    dolt sql -q "CREATE TABLE bitted (id int PRIMARY KEY, b bit)"
+    dolt sql -q "INSERT INTO bitted VALUES (1, 0), (3, 1)"
+
+    dolt table export bitted bitted.csv
+
+    run dolt table import -u bitted bitted.csv
```

Comment:
(minor) is it worth using `-cf` to force recreation of the table instead of updating it in place? Just wondering if the data could be skipped and this test still pass.

Reply:
This was modeling the customer issue flow